frontend/src/components/dialogs/SaveLinkDialog.tsx

```
@@ -0,0 +1,507 @@
+            >
+              <span className="material-symbols-outlined !text-[18px]">close</span>
+            </button>
+          </div>
+
+          <div className="relative px-5 pb-5 space-y-3 mt-3 z-10">
+            
+            {/* URL Input */}
+            <div className={`${styles.sectionContainer} space-y-1.5`}>
+              <div className="flex items-center gap-1.5">
+                <span className={`material-symbols-outlined !text-[12px] ${theme.accentPurple}`}>link</span>
+                <label className="block text-[10px] font-extrabold text-slate-500 uppercase tracking-widest">URL</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)